Source/SubtitleParser.cpp

```diff
@@ -0,0 +1,59 @@
+#include "pch.h"
+#include "SubtitleParser.h"
+#include "SubtitleParser.g.cpp"
+#include "FFmpegMediaSource.h"
+#include "MediaSourceConfig.h"
+
+namespace winrt::FFmpegInteropX::implementation
+{
+    winrt::Windows::Foundation::IAsyncOperation<winrt::FFmpegInteropX::SubtitleParser> SubtitleParser::ReadSubtitleAsync(winrt::Windows::Storage::Streams::IRandomAccessStream stream,
+        hstring streamName,
+        winrt::FFmpegInteropX::MediaSourceConfig config,
+        winrt::Windows::Media::Core::VideoStreamDescriptor videoDescriptor,
+        uint64_t windowId,
+        bool useHdr)
```

Comment:
Isn't this only used in internal APIs and carrying the result of the HDR detection (if auto, and otherwise the configured value).
Even when there's a reason for having the WindowId parameter, I don't think there's one for this param.